cmd/run.go

```diff
@@ -124,7 +135,11 @@ var RunCmd = &cobra.Command{
 					os.Exit(1)
 				}
 
-				daprRunning <- true
+				//if app exposes service, don't wait for daprd
```

Comment:
Can remove this if block: we still need to wait for daprd even if the app exposes a service. There's a small window between when Dapr detects the app is running and when it starts its gRPC and http server.

Reply:
cannot block app which expose services, since daprd initRuntime() will block wait for app service up, cause deadloop.